app/assets/stylesheets/home_styling.scss

```diff
@@ -0,0 +1,17 @@
+.container { position:relative; }
+.container img {
+  position: fixed;
```

Comment:
Properties should be ordered background-size, height, left, min-height, min-width, position, top, -moz-transform, -ms-transform, -webkit-transform, transform, width, z-index, z-index